server/src/app.ts

```diff
@@ -53,6 +56,12 @@ export class App {
 
 	private loadMiddleware(): void {
 		GlobalMiddleware.load(this.app);
+		if (this.config.state.docs) {
+			SwaggerMiddleware.load(this.app, {
```

Comment:
Kunnen we eigenlijk niet beter de docs package updaten? 🤔

Reply:
Dat kan inderdaad, ik stel voor dat we tijdens de ontwikkelingen op Recycle (of eventueel achteraf) kijken waar we packages van kunnen maken. Bijvoorbeeld error handling, paginatie, docs, mongodb, authenticatie, ...